package-lock.json

```diff
@@ -19,9 +20,13 @@
         "classnames": "^2.3.2",
         "commafy": "^0.0.6",
         "compact-object-deep": "^1.0.0",
+        "cookie": "^0.6.0",
```

Comment:
This can be removed since it is only using tiny-cookie

Reply:
It is actually the opposite. this is being used in the lambda to pull out the launchpad token.
The tiny cookie isn't being used in this code, but I have a follow on PR that will use the tiny cookie in the browser.